### DISCUSSION TITLE
"strict configuration" info page/description missing?

### DISCUSSION THREAD

**Discussion by mkosir** [typescript-eslint/typescript-eslint#6804] [Community Feedback]
Hey, just wondering usually I used this link to view info on [strict configuration](https://typescript-eslint.io/docs/linting/configs#strict), just wondering if it was removed on purpose?

It's also still shown in docs search results - https://typescript-eslint.io/search?q=strict
![image](https://user-images.githubusercontent.com/3073826/229123957-673e344f-ce69-4a93-83ed-ad1542467999.png)
Thanks

**Comment by JoshuaKGoldberg** [comment]
Ah, we changed our routing structure in #6138:

```diff
- https://typescript-eslint.io/docs/linting/configs#strict
+ https://typescript-eslint.io/linting/configs#strict
```

This looks like a bug in the search index.

@Josh-Cena forgive me, I think you've told me this before - do we know of a way to get Algolia to recreate the index? I can't find it in their web app if it exists.

**Reply by armano2** [reply]
we should add redirect as old links had `/docs/` in them and now they don't

**Reply by JoshuaKGoldberg** [reply]
Filed as a bug - #6805

Thanks for reporting @mkosir!

**Reply by mkosir** [reply]
@JoshuaKGoldberg thanks for quick response. I see [bug](https://github.com/typescript-eslint/typescript-eslint/issues/6805) is closed, but still can't find page [strict configuration](https://typescript-eslint.io/docs/linting/configs#strict) 🤔

**Reply by armano2** [reply]
issue is fixed for now in https://main--typescript-eslint.netlify.app/ and will be released with next release (it happens ~once per week)

https://main--typescript-eslint.netlify.app/docs/linting/configs#strict

**Reply by mkosir** [reply]
Of course thx, been too early Monday morning for me 🙃
